cmd/validator/flags/flags.go

```diff
@@ -109,25 +104,27 @@ var (
 		Usage: `Comma separated list of key value pairs to pass as gRPC headers for all gRPC calls.
 		Example: --grpc-headers=key=value`,
 	}
-	// GRPCGatewayHost specifies a gRPC gateway host for the validator client.
-	GRPCGatewayHost = &cli.StringFlag{
-		Name:  "grpc-gateway-host",
-		Usage: "Host on which the gateway server runs on.",
-		Value: DefaultGatewayHost,
-	}
-	// GRPCGatewayPort enables a gRPC gateway to be exposed for the validator client.
-	GRPCGatewayPort = &cli.IntFlag{
-		Name:  "grpc-gateway-port",
-		Usage: "Enables gRPC gateway for JSON requests.",
-		Value: 7500,
-	}
-	// GRPCGatewayCorsDomain serves preflight requests when serving gRPC JSON gateway.
-	GRPCGatewayCorsDomain = &cli.StringFlag{
-		Name: "grpc-gateway-corsdomain",
-		Usage: `Comma separated list of domains from which to accept cross origin requests (browser enforced).
-		This flag has no effect if not used with --grpc-gateway-port.
-`,
-		Value: "http://localhost:7500,http://127.0.0.1:7500,http://0.0.0.0:7500,http://localhost:4242,http://127.0.0.1:4242,http://localhost:4200,http://0.0.0.0:4242,http://127.0.0.1:4200,http://0.0.0.0:4200,http://localhost:3000,http://0.0.0.0:3000,http://127.0.0.1:3000"}
+	// HTTPServerHost specifies a HTTP server host for the validator client.
+	HTTPServerHost = &cli.StringFlag{
+		Name:    "http-host",
+		Usage:   "Host on which the HTTP server runs on.",
+		Value:   DefaultHTTPServerHost,
+		Aliases: []string{"grpc-gateway-host"},
+	}
+	// HTTPServerPort enables a HTTP server port to be exposed for the validator client.
+	HTTPServerPort = &cli.IntFlag{
+		Name:    "http-port",
+		Usage:   "Port on which the HTTP server runs on.",
+		Value:   7500,
```

Comment:
Should we have a variable for this: `DefaultHTTPServerPort` like we do for the host

Reply:
sure that seems like a good idea

Reply:
actually lets not do that, i looked at the references and it seems like it's just for a flag check. we don't need the default for anything else right now. we need a better strategy for these config values in general though ...